iozone/iozone_run.sh

```diff
@@ -645,8 +645,8 @@ do_test_actual()
 			else
 				let "file_size=$max_file_size/$4"
 				echo ================================================ >> ${iozone_output_file}
-				echo ${iozone_exe} -t $4 -i ${6} -+n -r ${page_size} -s${file_size}g -e -c -w -C ${iozone_args}-F ${5} >> ${iozone_output_file}
-				time taskset -c 0  ${iozone_exe} -t $4 -i ${6} -+n -r ${page_size} -s${file_size}g -e -c -w -C ${iozone_args} -F ${5} >> ${iozone_output_file}
+				echo ${iozone_exe} -t $4 -i ${6} -+n -r ${page_size} -s${file_size}g -c -w -C ${iozone_args}-F ${5} >> ${iozone_output_file}
```

Comment:
By removing the -e that means you can't count the fsync()/fflush() time in the timing results.  That's not an issue in throughput mode, but I think it'd affect the results in automatic mode.  That change in functionality isn't mentioned in the initial PR comment - is it intentional?

Reply:
Never mind, I see where you added it below instead.  Serves me right for commenting inline rather than reviewing in toto and then going back.